src/Site/views/languageforge/theme/default/page/home/index.html.twig

```diff
@@ -86,30 +77,20 @@
             </div>
         </section>
 
-        <div class="row">
-            <div class="6u 12u(narrower)">
-
+        <div class="row" id="more">
+            <div>
                 <section class="box special">
                     <span class="image featured"><img src="{{ baseDir }}/images/pic03.jpg" alt=""/></span>
-                    <h3>Help Improve Language Forge</h3>
-                    <p>You can discuss Language Forge, see recent improvements, suggest features and get community help
-                        on our SIL Software Community page.</p>
+                    <h3>Additional Resources</h3>
+                    <p>Thre are plenty of resources to get help and keep up with the latest features.</p>
                     <ul class="actions">
-                        <li><a href="https://community.software.sil.org/c/language-forge" class="button alt">Get Help, Discuss Features</a></li>
+                        <li><a target="_blank" href="https://community.software.sil.org/c/language-forge" class="button alt"><i class="fa fa-users"></i> Help and tips from our active community</a></li>
+                        <li><a target="_blank" href="https://community.software.sil.org/t/w/5454" class="button alt"><i class="fa fa-gift"></i> See what's new</a></li>
+                        <li><a target="_blank" href="https://www.youtube.com/playlist?list=PLJLUPwIFOI8d8lmQVAcBapyw87jCtmDNA" class="button alt"><i class="fa fa-youtube-play"></i> Videos</a></li>
+                        <li><a target="_blank" href="https://github.com/sillsdev/web-languageforge/wiki/Known-Issues-and-Limitations" class="button alt"><i class="fa fa-exclamation-triangle"></i> Known issues and limitations</a></li>
+                        <li><a target="_blank" href="mailto:issues@languageforge.org" class="button alt"><i class="fa fa-envelope"></i> Report a problem (email issues@languageforge.org)</a></li>
```

Comment:
This would make a it just a tad prettier.
```suggestion
                        <li><a target="_blank" href="mailto:issues@languageforge.org" class="button alt"><i class="fa fa-envelope"></i> Report a problem (issues@languageforge.org)</a></li>
```